modules/virt-defining-storageclass-in-cdi.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass-in-cdi_{context}"]
+= Defining a storage class in the CDI object
+
+Define a storage class in the `CDI` object  to dynamically provision scratch space for CDI operations.
+
+.Procedure
+
+* Use the `oc` client to edit the `CDI` object and edit the value of the `spec.config.scratchSpaceStorageClass` field to match a storage class in the cluster.
++
+[source,terminal]
+----
+$ oc edit cdi
+----
++
+[source,yaml]
+----
+API Version: cdi.kubevirt.io/v1beta1
```

Comment:
Unless I am trippin':

s/API Version/apiVersion

Reply:
You aint trippin :)